tests/issues/issue-9/issue-9.test.ts

```diff
@@ -11,6 +11,8 @@ describe('Issues', () => {
 			parser: 'pug',
 			plugins: [plugin],
 
+			// @ts-expect-error
+			pugFramework: 'angular',
```

Comment:
_cannot add comment to line 7 ^^_

`should escape quotes correctly in angular`